src/renderer/pages/dashboard.vue

```
@@ -63,7 +63,18 @@
 
         <!-- Empty State -->
         <div v-else-if="projects.length === 0" class="py-16 text-center">
-          <p class="text-muted-foreground mb-4">{{ t("dashboard.empty") }}</p>
+          <div class="space-y-4">
+            <h2 class="text-foreground text-2xl font-bold">{{ t("dashboard.empty.welcome") }}</h2>
+            <div class="text-muted-foreground space-y-2 max-w-2xl mx-auto text-left">
```

ESLint

[error] Replace `space-y-2·max-w-2xl·mx-auto` with `mx-auto·max-w-2xl·space-y-2`

(prettier/prettier)